meson.build

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 Endless Mobile, Inc.
+
+project('contentfeed', 'c', version: '0.0.0', license: 'LGPL2.1+',
+    meson_version: '>= 0.42.0',
+    default_options: ['c_std=c99', 'warning_level=3'])
+
+gnome = import('gnome')
+pkg = import('pkgconfig')
+i18n = import('i18n')
+
+version_components = meson.project_version().split('.')
+api_version = version_components[0]
+api_name = '@0@-@1@'.format(meson.project_name(), api_version)
+libtool_version = '0.0.0'
+namespace_name = 'EosDiscoveryFeed'
```

Comment:
This should be changed

Reply:
See my other comment - I'm not sure I want to introduce that much churn into this branch :/